tests/conftest.py

```diff
@@ -25,11 +29,28 @@
 from .factories import applications as application_factories
 
 
+# set up tracing for tests
+provider = TracerProvider()
+trace.set_tracer_provider(provider)
+test_exporter = InMemorySpanExporter()
+provider.add_span_processor(SimpleSpanProcessor(test_exporter))
```

Comment:
Should these be in a session scoped fixture?  I'm not sure if that would have any meaningful difference…